src/FastTags/LensBlaze.hs

```diff
@@ -0,0 +1,64 @@
+----------------------------------------------------------------------------
+-- |
+-- Module      :  FastTags.LensBlaze
```

Comment:
Can you add a little bit about what and why?  I gather it's meant to be a mini-lens implementation, but is it supposed to be faster the other lenses?  Or avoiding the heavy dependency?  Or something else?

Reply:
It's mostly to avoid any dependencies, fully inline everything, and we'll need to define lenses that keep integers within biger integers anyway.